front/app/components/Form/Components/Controls/MultiSelectCheckboxControl.tsx

```diff
@@ -44,14 +44,38 @@ const MultiSelectCheckboxControl = ({
   visible,
 }: ControlProps) => {
   const [didBlur, setDidBlur] = useState(false);
+  const { formatMessage } = useIntl();
   const answerNotPublic = uischema.options?.answer_visible_to === 'admins';
   const options = getOptions(schema, 'multi');
   const dataArray = Array.isArray(data) ? data : [];
 
+  const maxItems = schema.maxItems;
+  const minItems = schema.minItems;
+
   if (!visible) {
     return null;
   }
 
+  const getInstructionMessage = () => {
+    if (minItems && maxItems) {
```

Comment:
> Also if minItems <= 1 && maxItems === options?.length the copy 'You can choose as many as you like' does not seem correct. You cannot choose as many as you like, because you have to choose a minimum

Hmm yes I see what you're saying - though this was the behaviour we had before so I didn't want to change that :thinking: Regardless of if the question was required or optional, we would show "Choose as many as you like". I can change that though so it shows "Select between 1 and 4" then if you think that makes more sense :+1:

Reply:
> "Select between 1 and 4" then if you think that makes more sense
That would be best idd!